crates/bevy_ecs/src/world/mod.rs

```diff
@@ -579,6 +580,7 @@ impl World {
 
     /// Returns an iterator of entities that had components with the given `component_id` removed
     /// since the last call to [`World::clear_trackers`].
+    // TODO: return `std::iter::Copied` instead
```

Comment:
I think we should return a `std::iter::Copied` here but I don't know if it would be a breaking change or not